clientServiceClient/clientServiceClient.go

```diff
@@ -76,7 +76,7 @@ func (c *ClientServiceClient) AdminToggleClientEnabled(ctx context.Context, para
 // Register registers a client.
 func (c *ClientServiceClient) Register(ctx context.Context, params ClientRegisterRequest) (*ClientRegisterResponse, error) {
 	var result *ClientRegisterResponse
-	path := c.Host + "/" + "v1/account/e3db/clients/register"
+	path := c.Host + "/" + ClientServiceBasePath
```

Comment:
this should be its own PR, reminder 
`InternalRegister` -- client service 
`Register`-- account